### PR TITLE
Document that "same" is inconsistent across backends with strides!=1

### PR DESCRIPTION
**Pull request by ozabluda**
As described in https://github.com/keras-team/keras/pull/9473#issuecomment-372166860

### CODE REVIEW

File: keras/layers/convolutional.py [hunk 1]
```diff
@@ -373,6 +373,9 @@ class Conv2D(_Conv):
             Specifying any stride value != 1 is incompatible with specifying
             any `dilation_rate` value != 1.
         padding: one of `"valid"` or `"same"` (case-insensitive).
+            Note that `"same"` is inconsistent across backends with `strides` != 1,
+            as described at
+            https://github.com/keras-team/keras/pull/9473#issuecomment-372166860
```
Comment (fchollet) [inline keras/layers/convolutional.py]: Please format the link ("as described [here](...)")

Comment (fchollet) [reply keras/layers/convolutional.py]: Also say "may be slightly inconsistent"

since 1) it is not necessarily the case depending on various parameters, 2) the difference is minor and generally unimportant

**Comment by ozabluda** [comment]
Comments addressed.

For the record, TF convolutions with `padding="same", strides !=1` are described here:
https://www.tensorflow.org/api_guides/python/nn#Convolution

Cross-backend Keras code which clarifies TF behavior is here:
https://github.com/tensorflow/tensorflow/issues/17654
